docs/develop/go/observability.mdx

```diff
@@ -197,12 +197,26 @@ OpenTracing and OpenTelemetry are natively supported by [Jaeger](https://www.jae
 For more information on configuring and using tracing, see the documentation provided by [OpenTracing](https://opentracing.io), [OpenTelemetry](https://opentelemetry.io/), and [Datadog](https://docs.datadoghq.com/tracing/).
 
 ## Log from a Workflow {#logging}
-
 **How to log from a Workflow using the Go SDK.**
 
 Send logs and errors to a logging service, so that when things go wrong, you can see what happened.
 
-The SDK core uses `WARN` for its default logging level.
+Loggers create an audit trail and capture information about your Workflow's operation.
+An appropriate logging level depends on your specific needs.
+During development or troubleshooting, you might use debug or even trace.
```

Comment:
On the preview, it seems like debug, trace, info, warn should have command ticks on it. It looks a little odd without it.